src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -658,6 +659,11 @@ protected override void RegisterCallbacks()
                 FramebufferResize?.Invoke(new(width, height));
             };
 
+            _onRefresh = (window) =>
+            {
+                Refresh?.Invoke();
```

Comment:
Another feasible option is to keep the API surface the same and call the  Update/Render methods here, perhaps with a zero update value to indicate an inter-frame update.

Reply:
I think I'm leaning more towards this after seeing the latest changes (per the above comment)